pages/match.tsx

```diff
@@ -1,5 +1,19 @@
+import { useEffect, useState } from "react";
 import MatchBoardList from "../components/MatchBoardList";
+import { MatchData } from "../types/matchTypes";
+import { getData } from "../util/getData";
 
 export default function Match() {
-  return <MatchBoardList />;
+  const [matchData, setMatchData] = useState<MatchData | null>(null);
+  useEffect(() => {
+    (async () => {
+      try {
+        const data = await getData(`/pingpong/match/table/${1}?type=single`);
```

Comment:
```suggestion
        const data = await getData(`/pingpong/match/tables/${1}?type=single`);
```
url 고쳐주시면 좋을 것 같아요!!